### ISSUE TITLE
Review seems to assume the base branch is `main`

### ISSUE THREAD

**Issue by dividedmind** [getappmap/appmap-js#2129]
I tried the following in our vscode-appland repository:
![obraz](https://github.com/user-attachments/assets/3bd3d9ee-7c07-4e34-92a6-25858ab9d43f)

A couple of notes:
- it seems to assume the base is `main`,
- it runs `git` with system locale. If we're consuming `git` output in code, we should probably run it with `C` locale and/or try to use low-level commands with guaranteed stable output (though this might not apply to error messages — however maybe we should verify first to make sure a command won't fail),
- it also appears hung — it probably crashed; I'd expect some kind of error recovery.

**Comment by kgilpin** [comment]
You don’t have to guess - there is documentation 

https://appmap.io/docs/navie-reference/navie-commands.html#review

**Comment by kgilpin** [comment]
I am closing this because the behavior re: main branch is as documented. 

You can open separate issues if you believe you have found a bug. 

Confirm the bug and provide a reproduction if you want to report something. As an appmap developer, I don’t want to see guesses - eg “it appears hung”.

**Comment by dividedmind** [comment]
I just noticed that while doing something else and thought I'd open a bug report because from the point of view of a user it definitely looks broken. Figured it'd be worth it to track this and flesh this out later. It does appear hung to the user, it's a statement of fact, not a guess; the suggest box keeps spinning. Are you saying there's no value in documenting these kinds of UX bugs for tracking even if they don't start out fully fleshed out?

(BTW, you closed this as completed; I think you meant to close as invalid/wontfix.)
